### ISSUE TITLE
DataSchema definition should be from the JSON Schema vocab, not from the TD vocab

### ISSUE THREAD

**Issue by vcharpenay** [w3c/wot-thing-description#186]
See `validation/td-validation.ttl`

**Comment by mkovatsc** [comment]
You mean
```
@prefix tdschema: <http://www.w3.org/ns/td/schema#> .
@prefix tdsec: <http://www.w3.org/ns/td/security#> .
```
should be
```
@prefix tdschema: <http://www.w3.org/ns/json-schema#> .
@prefix tdsec: <http://www.w3.org/ns/wot-security#> .
```
?

Actually, the prefixes should probably also rather be `jsons` and `wotsec`.

**Comment by vcharpenay** [comment]
This part has been fixed in #180.

This issue is rather a reminder to myself that there are some errors in the SHACL definitions. In particular, the target of `:DataSchemaShape` should be `jsonschema:DataSchema` instead of `td:DataSchema`.
